models/classes/pack/ItemPack.php

```diff
@@ -133,37 +133,42 @@ public function getData()
      *
      * @param string $type the assets type, one of those who are supported.
      * @param array $assets the list of assets' URL to load
+     * @param StorageDirectory|null $publicDirectory
      *
-     * @throw InvalidArgumentException
+     * @throws ExceptionMissingEncoder
+     * @throws FileNotFoundException
      */
-    public function setAssets(string $type, $assets): void
+    public function setAssets(string $type, $assets, ?StorageDirectory $publicDirectory = null): void
```

Comment:
I don't really see which part of the codebase calls this method with a third argument.

Reply:
@wazelin 
https://github.com/oat-sa/extension-tao-itemqti/blob/9f8cb729ade4d7b9346de9d143178b2b019f66d6/model/pack/QtiItemPacker.php#L123

Reply:
issue introduced here https://github.com/oat-sa/extension-tao-item/pull/410